CaT/src/Filter/Navigator.php

```diff
@@ -107,6 +107,6 @@ protected function getItemByPath($path, $tmp) {
 	}
 
 	public function path() {
-		return implode("_",$this->path);
+		return is_array($this->path) ? implode("_",$this->path) : null;
```

Comment:
@shecken 
Bitte einmal angucken und bewerten. Hier hat sich klamm und heimlich der Rückgabetyp der Methode von string zu string oder null geändert und die Konsistenzbedingungen für den Navigator ebenfalls (von is_array($this->path) zu $this->path kann irgendwas sein). Ich kann gerade nicht einschätzen, ob das so richtig ist oder ob $this->path initialisiert werden müsste.

Bitte nachher die Methode mit Doku versehen, und dann am besten noch den ganzen Navigator, aber in eigenem PR. Danke!